routes/baseDBMethods.js

```diff
@@ -22,6 +24,35 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function proccessDatesRangeParameters(ctx) {
+  const startDate = parseInt(ctx.query.startDate, 10) || null;
+  const endDate = parseInt(ctx.query.endDate, 10) || null;
+
+  const datesFilter = {};
+
+  if ((startDate != null) && (endDate != null) && (startDate > endDate)) {
+    throw new RangeError('Query parameter \'startDate\' can\'t be bigger than Query parameter \'endDate\'');
+  } else if ((startDate != null) || (endDate != null)) {
+    datesFilter.date = {};
+
+    if (startDate != null) {
+      if (startDate < 1) {
+        throw new RangeError('Query parameter \'startDate\' must be a positive number');
+      }
+      datesFilter.date.$gte = moment(startDate).tz('Israel').format();
```

Comment:
Do you think that specifying the format explicitly is preferable? (For your own decision)

Reply:
This give my the format MongoDB use, which is ISO Format, I think it should be ok.